src/nrf_802154_trx.c

```diff
@@ -112,6 +112,9 @@
 #define SHORTS_RX_ACK           (NRF_RADIO_SHORT_ADDRESS_RSSISTART_MASK | \
                                  NRF_RADIO_SHORT_END_DISABLE_MASK)
 
+#define SHORTS_MOD_CARRIER      (NRF_RADIO_SHORT_READY_START_MASK | \
```

Comment:
Just wondering - do you know why it's `NRF_RADIO_SHORT_READY` and not `NRF_RADIO_SHORT_TXREADY`?

Reply:
Good catch!
I might have suggested it in the task description, by mistake. Both shorts should work fine in this case, but I think we usually use `TXREADY` or `RXREADY` instead of `READY` in the driver.